iot/http_example/cloudiot_http_example.js

```diff
@@ -182,6 +201,21 @@ function publishAsync (authToken, messageCount, numMessages) {
 // [START iot_http_getconfig]
 function getConfig (authToken, version) {
   console.log(`Getting config from URL: ${urlBase}`);
+
+  // Custom backoff functions
+  function retryNotSuccess (err, response, body) {
```

Comment:
Is there a library that comes with these built-in? (Perhaps [nodejs-repo-tools?](https://github.com/GoogleCloudPlatform/nodejs-repo-tools))

Reply:
Done, switched to recommended library that uses built-in exponential backoff.